glide.yaml

```diff
@@ -18,10 +18,21 @@ import:
   version: 2152b45fa28a361beba9aab0885972323a444e28
 - package: github.com/golang/glog
 - package: github.com/vishvananda/netlink
+  version: 6e453822d85ef5721799774b654d4d02fed62afb
+- package: k8s.io/apimachinery
+  version: 103fd098999dc9c0c88536f5c9ad2e5da39373ae
+  subpackages:
+  - pkg/apis/meta/v1
+- package: k8s.io/api
+  version: 072894a440bdee3a891dea811fe42902311cd2a3
+  subpackages:
+  - core/v1
 - package: k8s.io/client-go
-  version: 4a3ab2f5be5177366f8206fd79ce55ca80e417fa
+  version: 7d04d0e2a0a1a4d4a1cd6baa432a2301492e4e65
   subpackages:
   - kubernetes
   - tools/clientcmd
+  - util/retry
```

Comment:
Hi @dcbw ,

I am trying this glide update issue for nearly 10 hours :confused: from yesterday, no luck :disappointed: (It must be a small issue or silly mistake from my side, I am missing something here) . 

I am getting exactly same error, 
Setting version for golang.org/x/time to f51c12702a4d776e4c1fa9b0fabab841babae631.
[ERROR] Error scanning k8s.io/client-go/pkg/api/v1: open /root/.glide/cache/src/https-k8s.io-client-go/pkg/api/v1: no such file or directory

only 3 package - k8s.io/apimachinery, k8s.io/api and k8s.io/client-go is used in our project.

Please check the concept correctness for updating the pod in Multus.

Once I solved this issue. I can quickly compile the code and merge it.

Reply:
I suppose that glide does 'deep update', including Godeps in vendoring pkgs. `glide update --quick` does quick update, just in Glide.yaml so we don't see any error.